web/src/rtl.ts

```diff
@@ -21,26 +21,33 @@ import _ from "lodash";
  * @param {string} raw
  * @returns {number[]}
  */
-function convert_from_raw(digits, part_length, raw) {
-    const result = [];
+function convert_from_raw(digits: string, part_length: number, raw: string): number[] {
+    const result: number[] = [];
```

Comment:
Is the type definition required here? 
Since only `number`s are being pushed into the `result` array, TS can infer this array as `number[]` itself.

Reply:
@sbansal1999 yes that's true. I think it's best practice to annotate the type of of everything beforehand instead of relying on type inference. If you think I should remove it lemme know.

Reply:
Yes, it's good to annotate things but if we were to annotate everything, then we don't make use of type inference (which is where the advantage of TypeScript lies). @PIG208 thoughts on this?